package.json

```diff
@@ -18,21 +18,25 @@
     "name": "LBRY Inc.",
     "email": "hello@lbry.io"
   },
-  "main": "./dist/main/main.js",
+  "main": "./dist/electron/main.js",
   "scripts": {
     "compile:electron": "webpack --progress --config webpack.electron.config.js",
     "compile:web": "webpack --progress --config webpack.web.config.js",
     "compile": "yarn compile:electron && yarn compile:web",
     "build": "yarn compile && electron-builder build",
     "build:dir": "yarn build -- --dir -c.compression=store -c.mac.identity=null",
-    "dev:electron": "yarn compile:electron && electron ./dist/electron/main/bundle.js",
+    "dev:electron": "webpack-dev-server --hot --progress --config webpack.electron.config.js",
+    "dev:web": "webpack-dev-server --hot --progress --config webpack.web.config.js",
     "dev:internal-apis": "LBRY_API_URL='http://localhost:8080' yarn dev:electron",
     "lint": "eslint 'src/**/*.{js,jsx}' --fix && flow",
     "format": "prettier 'src/**/*.{js,jsx,scss,json}' --write",
     "flow-defs": "flow-typed install",
     "precommit": "lint-staged",
     "preinstall": "yarn cache clean lbry-redux && yarn cache clean lbryinc",
-    "postinstall": "electron-builder install-app-deps && node build/downloadDaemon.js"
+    "postinstall": "electron-builder install-app-deps && node build/downloadDaemon.js",
+    "run:electron": "electron ./dist/electron/main.js",
+    "pack": "electron-builder --dir",
+    "dist": "electron-builder"
```

Comment:
Do we need this if we have `build` and `build:dir`?

Reply:
Don't need either, I put these here both for convenience and testing.
Can remove, I left them thinking that can be nice to have locally